app/schemas/com.nextcloud.talk.data.source.local.TalkDatabase/11.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
   "database": {
     "version": 11,
```

Comment:
the databse version must be bumped and a database migration must be added to alter the affected table

Reply:
this was merged too fast. there should have been a 12.json file instead to modify 11.json
should be fixed with https://github.com/nextcloud/talk-android/pull/4417